yarn.lock

```diff
@@ -792,6 +792,90 @@
     "@jridgewell/resolve-uri" "3.1.0"
     "@jridgewell/sourcemap-codec" "1.4.14"
 
+"@node-rs/xxhash-android-arm-eabi@1.4.2":
+  version "1.4.2"
+  resolved "https://registry.yarnpkg.com/@node-rs/xxhash-android-arm-eabi/-/xxhash-android-arm-eabi-1.4.2.tgz#936b6b9b78035e1d136dd7bdf8296ad31fe1f4af"
+  integrity sha512-6aonl3wNY/axTiW9jSz+Z2RduP2xCZTgX1M1ur8uCJcUVv/kkv+dEMFb3o/k19qBGntPWvGXZTj2mYO6onDr/g==
+
```

Comment:
are these yarn lock changes expected?

Reply:
let me regenerate one

Reply:
@ps48, I just revert back. But the issue is when I run it on my local under the dashboards core boot strap, it will help me to update it, so it seems like the yarn lock file is out of date.

Reply:
I think we should also update that too. Do you want me to do it in this PR or create a new one?

Reply:
Transferring some discussion between me and @mengweieric here. Since original change in yarn lock in the previous commit was coming from the local setup of `yarn osd bootstrap` and it is irrelevant with the `babel` change itself so we can just revert that and make this PR clean. So I just reverted.